### ISSUE TITLE
Group properties on App properties Modal

### ISSUE THREAD

**Issue by oodamien** [spring-cloud/spring-cloud-dataflow-ui#1757]

**Comment by dturanski** [comment]
Related to https://github.com/spring-cloud/spring-cloud-dataflow/issues/4188

**Comment by dturanski** [comment]
@oodamien - Grouping looks great. Some properties are missing, like for `dataflow-tasklauncher` https://github.com/spring-cloud/spring-cloud-dataflow/blob/main/spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-sink/src/main/resources/META-INF/dataflow-configuration-metadata.properties . Missing `TriggerProperties` and `TasklauncherFunctionProperties` . I checked the metadata and TriggerProperties are present.  The TaskLauncherFunctionProperties are not, but I did a PR on SCDF to fix that.
